SWLOR.Game.Server/Conversation/BaseManagementTool.cs

```diff
@@ -62,19 +65,26 @@ public override PlayerDialog SetUp(NWPlayer player)
                 "75 degrees",
                 "90 degrees",
                 "180 degrees");
-            
-
+            DialogPage renamePage = new DialogPage("Type a name, Once you are done select confirm.",
```

Comment:
Grammar issue here. Maybe use "Type a name into the chat box. Once you are done select confirm."